pkg/sloop/server/internal/config/config_test.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"testing"
+	"encoding/json"
+	"github.com/ghodss/yaml"
+)
+
+func Test_loadFromJSONFile(t *testing.T) {
+	var expectedconfig SloopConfig
+	configfilename:="testconfig.json"
+	configFile, err := ioutil.ReadFile(configfilename)
+	err = json.Unmarshal(configFile, &expectedconfig)
+
+	out_config,err := loadFromFile(configfilename)
+
+	assert.Nil(t,err)
+	assert.Equal(t,out_config,&expectedconfig)
+}
+
```

Comment:
Also add test cases for error cases like file not available, errors in marshalling etc

Reply:
done